src/main/java/LiUni/TwitterStatus.java

```diff
@@ -0,0 +1,79 @@
+package LiUni;
+
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+import twitter4j.StatusUpdate;
+import twitter4j.Status;
+
+import java.net.URL;
+
+
+public class TwitterStatus {
+
+    private static final int TWITTER_CHAR_MAX = 280;
+    private static final int TWITTER_URL_LEN = 22;
+    private static Twitter twitter = TwitterFactory.getSingleton();
+
+    public void postStatus(String text) {
+        if (textErrorCheck(text)) {
+            StatusUpdate newStatus = new StatusUpdate(text);
+            try {
+                Status status = twitter.updateStatus(newStatus);
+                System.out.println("Successfully updated status to [" + status.getText() + "].");
+            }
+            catch (TwitterException e) {
+                System.out.println("Exception occurred. " + e.getErrorMessage());
+                System.out.println(e.getExceptionCode());
+            }
+        }
+        else {
+            System.out.println("\n\nAn error occurred. Unable to post your tweet [" + text + "]. Sorry!");
+        }
+    }
+
+    // True = No errors; False = Error occurred
+    private boolean textErrorCheck(String text) {
+        char[] convertedArr = text.toCharArray();
+
+        //Twitter shortens URLs to TWITTER_URL_LEN characters
+        boolean containsURL = statusContainsUrl(text);
+        int statusLength = convertedArr.length;
+        if (containsURL) {
+            //Handle URLs shorter than TWITTER_URL_LEN
+            int urlLen = getUrl(text).length();
+            if (urlLen < TWITTER_URL_LEN) {
+                statusLength -= urlLen;
+            }
+            else {
+                statusLength -= TWITTER_URL_LEN;
+            }
+        }
+
+        if (statusLength > TWITTER_CHAR_MAX) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private static boolean statusContainsUrl(String text) {
```

Comment:
why is this function static? also is there a reason why it is always defaulted to false?